gh_pages/conf.py

```diff
@@ -58,7 +58,7 @@
 
 # General information about the project.
 project = u'Industrial Training'
-copyright = u'2020, ROS-Industrial'
+copyright = u'2025, ROS-Industrial'
```

Comment:
I've tested the following on Ubuntu 20 and Ubuntu 22 and it works to automatically generate the current date and extract the year to apply. Not sure if that's what we want.

```python
import datetime
copyright = u'{date}, ROS-Industrial'.format(date = datetime.date.today().year)
```

Reply:
Seems reasonable to me